docs/tutorials/embedded/wine_detector.md

```diff
@@ -10,30 +10,29 @@
 
 If there is another similar tutorial that's more appropriate for another audience, direct the reader there with a linked reference. -->
 
-### What's In This Tutorial?
-In this demo we will walk through the steps to setup a real-time object classifier, powered by MxNet running entirely on a Raspberry Pi 3. We will then work through connecting this local Raspberry Pi model to the AWS cloud via the IoT service, letting you easily and reliablely get realtime updates from the device.
+## What's In This Tutorial?
+In this demo we will walk through the steps to set up a real-time object classifier, powered by MxNet running entirely on a Raspberry Pi 3. We will then work through connecting this local Raspberry Pi model to the AWS cloud via the IoT service, letting you easily and reliably get real-time updates from the device.
 
-* The primary goal of this Tutorial is to introduce developers who work with the Raspberry Pi or similar embedded ARM-based devices with a guide to compiling MxNet for those devices as well as running a non-trival, pre-trained deep network model on them. 
+* The primary goal of this tutorial is to provide developers who work with the Raspberry Pi or similar embedded ARM-based devices with a guide to compiling MxNet for those devices, as well as running a non-trivial, pre-trained deep network model.
```

Comment:
This tutorial shows developers who work with the Raspberry Pi or similar embedded ARM-based devices how to compile MXNet for those devices and running a pretrained deep network model.